source/reference/verification/embedded.txt

```diff
@@ -0,0 +1,72 @@
+
+.. _c2c-embedded-verifier:
+
+Verify with Embedded Verifier
+=============================
+
+.. versionadded:: 1.8
+
+   ``mongosync`` embeds a verifier to run a series of checks against the source
```

Comment:
Avoid "embeds". Also, should we assume "verifier" is something the reader will understand?

Reply:
Mitigated